### ISSUE TITLE
Reading parameters when running ascets_resources.R

### ISSUE THREAD

**Issue by ariellearabov** [beroukhim-lab/ascets#7]
Hi, 

When running the ascets_resources.R file I encountered that the min_boc parameter is read to be the one with the -t flag and not the -m as specified in the README. Is this a mistake? 

<img width="911" alt="Screen Shot 2023-10-24 at 11 29 59" src="https://github.com/beroukhim-lab/ascets/assets/92447675/456c3dc5-29db-4b9d-a451-36fea871b0d3">

Thanks in advance,
Arielle.

**Comment by lfspurr** [comment]
Hi Arielle,

Thank you very much for bringing this to my attention. You are absolutely correct, this was an error. The code is now fixed to use the correct parameter. I recommend rerunning any analyses prior to this version. 

However, as a test, I have compared our calls of the MSK-IMPACT data from 2017 (a subset of which was used in our manuscript here: https://www.nature.com/articles/s41588-022-01235-4), with those made using the exact same parameters (-t = 0.1). This would constitute the likely most extreme case where the BOC parameter was effectively changed from (an erroneous) 0.1 to 0.5. We have found very few differences in the calls and aneuploidy scores (both versions are attached). Across all 448,499 chromosome arms from 10,939 samples, only 116 arms (from 112 samples) were affected by this error.

Apologies and thank you again for bringing this to our attention. Feel free to reach out with any further questions.

Best,
Liam

[parameter_error_benchmark.csv](https://github.com/beroukhim-lab/ascets/files/13132256/parameter_error_benchmark.csv)
[parameter_error_arm_calls.csv](https://github.com/beroukhim-lab/ascets/files/13132621/parameter_error_arm_calls.csv)
